addons/exact-count/forum.js

```diff
@@ -2,10 +2,13 @@ export default async function ({ addon, global, console }) {
   while (true) {
     let userbox = await addon.tab.waitForElement(".postleft dl", { markAsSeen: true });
     if (userbox.childNodes[6] && userbox.childNodes[6].nodeValue.includes("+")) {
+      const postCountReal = Number(/(\d\d?\d?\d?)\+? posts?/.exec(userbox.childNodes[6].nodeValue)[1]);
```

Comment:
`posts` would need to be translated...

Reply:
Most of the forums isn't translated.

Reply:
> 
> 
> `posts` would need to be translated...

Its never translated. I even checked by switching to a different language